metadata.yaml

```diff
@@ -12,7 +12,7 @@ resources:
     type: oci-image
     description: 'Backing OCI image'
     auto-fetch: true
-    upstream-source: docker.io/kubeflownotebookswg/centraldashboard:v1.8.0
+    upstream-source: docker.io/kubeflownotebookswg/centraldashboard:v1.9.0-rc.0
```

Comment:
regarding this, I didn't take a look at the manifests but I see this change in upstream image command. Should we update the charm's command as well? 
[1.8.0](https://github.com/kubeflow/kubeflow/blob/v1.8-branch/components/centraldashboard/Dockerfile) -> [1.9.0-rc.0](https://github.com/kubeflow/kubeflow/blob/v1.9.0-rc.0/components/centraldashboard/Dockerfile#L31)

Reply:
good point, updated
but if we will integrate the rock later then we will not use tini

Reply:
I think that it is worth adding a comment just so we do not forget about changing this. Something like "remove tini when using the rock"

Reply:
added

Reply:
hmm what about adding the comment in the metadata.yaml? I feel like this is the place that guarantees we 'll see it when updating an image. I mean, we should in general take a look at the pebble layer and switching to rocks, but one could always forget about that.

Reply:
added a comment in metadata too